packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -279,6 +286,84 @@ def draw_image(self, gc, x, y, im, transform=None):
         self.ctx.drawImage(in_memory_canvas, x, y, w, h)
         self.ctx.restore()
 
+    def _get_font(self, prop):
+        fname = findfont(prop)
+        font_name = fname[fname.rfind('/')+1:]
+        font = FT2Font(str(fname))
+        font.clear()
+        font.set_size(prop.get_size_in_points(), self.dpi)
+        return font, font_name
```

Comment:
Maybe rename `font_name` to `file_name`.  A font name would be something like "Times New Roman".  The file name would be "times.ttf".